mdoc/Test/DocTest-v1.cs

```diff
@@ -649,6 +649,13 @@ public interface IFoo<T> {
 		T Method <U> (T t, U u);
 	}
 
+	/// <typeparam name="NGparam">Non-generic param</typeparam>
+	/// <remarks><c>T:Mono.DocTest.CompoundConstraints</c>.</remarks>
+	public class CompoundConstraints<NGparam> where NGparam : class, new()
+    {
+
```

Comment:
Please add a member that includes a compound type constraint as well. This way we can make sure that this works for member signatures as well

Reply:
New member with compound type constraint added